esmvalcore/preprocessor/_area.py

```diff
@@ -59,14 +59,16 @@ def extract_region(cube, start_longitude, end_longitude, start_latitude,
     if cube.coord('latitude').ndim == 1:
         region_subset = cube.intersection(
             longitude=(start_longitude, end_longitude),
-            latitude=(start_latitude, end_latitude))
+            latitude=(start_latitude, end_latitude),
+            ignore_bounds=True,
+        )
```

Comment:
it'd be useful if you added a comment that `ignore_bounds` defaults to `False` - I had to look up the iris documentation to be 100% sure the default is false

Reply:
Ok